vnext/Shared/Threading/BatchingQueueThread.h

```diff
@@ -3,11 +3,41 @@
 
 #pragma once
 
+#include <ReactCommon/CallInvoker.h>
 #include <Shared/BatchingMessageQueueThread.h>
 #include <thread>
 
+namespace facebook::react {
+class Instance;
+}
+
 namespace react::uwp {
 
+struct BatchingQueueThread;
```

Comment:
>BatchingQueueThread [](start = 7, length = 19)

nit: is fwd decl really needed? It is not used by BatchingQueueCallInvoker...

Reply:
Thanks - Left over from earlier iterations of the fix.